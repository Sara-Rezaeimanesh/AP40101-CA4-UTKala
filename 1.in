POST signup ? username ali password 123123 role seller address Karaj
POST login ? username ali password 123123
