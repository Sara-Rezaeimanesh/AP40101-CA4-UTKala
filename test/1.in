POST signup ? username karim password 123123 role seller address Karaj
POST add_item ? name shokolat price 100 category chocolates quantity 10 refund 50
POST add_item ? name milk price 120 category dairy quantity 60
POST logout
POST signup ? username ali password 123123 role buyer address Karaj
POST login ? username ali password 123123
POST increase_credit ? amount 10000
POST buy_item ? id 1 count 400 city Karaj
POST buy_item ? id 2 count 5 city Karaj
POST buy_item ? id 3 count 5 city Karaj
GET wallet_balance
GET list_purchased
