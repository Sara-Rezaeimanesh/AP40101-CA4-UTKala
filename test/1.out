OK
id : 1
OK
id : 2
OK
OK
OK
OK
OK
Bad Request
total_cost : 2600
OK
Not Found
Bad Request
wallet balance : 7400
OK
id : 1
date : Tue Jan 10 19:51:20 2023
seller : karim
item_id : 2
item_name : milk
item_price : 120
purchased_count : 5
delivery_cost : 2000
total_cost : 600

OK
